ansible/roles/automation/nightly_pr/defaults/main.yml

```diff
@@ -51,3 +51,17 @@ nightly_jobs:
     flow: "pki"
     branch: "master"
     prci_config: "nightly_master_pki.yaml"
+  - name: testing_389ds
+    weekdays: "6"
+    hour: "20"
+    minute: "00"
+    flow: "389ds"
+    branch: "master"
+    prci_config: "nightly_master_389ds.yaml"
+  - name: testing_updates_testing
+    weekdays: "6"
+    hour: "21"
```

Comment:
only 1 hour of difference with testing_389ds?

Reply:
That's on Saturday, so I'll update it to `10`, ok?

Reply:
ok! yes please. It is important that we don't schedule two or more at same time, otherwise will fail.